CHANGELOG.md

```diff
@@ -1,18 +1,21 @@
 **New Features and Improvements**
 
 * Mod Support:
-  - Added support for ID24 `SKYDEFS` and intermissions.
-  - Added support for ID24 `SBARDEF` with additional widescreen layout and widgets. Remove Boom HUD.
-  - Added support for PNG graphics, textures, flats and sprites. Automatically convert PNG palette. Convert 32-bit PNG to 8-bit with color degradation (not recommended).
+  - ID24 features:
+    - `SBARDEF`, with additional widescreen layout and widgets; consequently, removed Boom HUD.
+    - `SKYDEFS`.
+    - `TRAKINFO` (remixed soundtrack from "Doom + Doom II").
```

Comment:
`TRAKINFO` is not part of the ID24 spec.

Reply:
My bad, fixed in https://github.com/fabiangreffrath/woof/pull/2026/commits/0afe4b73121ce40225da7d8b9285509b7b9cf776.